package.json

```diff
@@ -57,31 +57,29 @@
   },
   "dependencies": {
     "@babel/core": "^7.16.0",
-    "@babel/polyfill": "^7.12.1",
     "@babel/preset-env": "^7.16.4",
-    "@comunica/config-query-sparql": "^2.0.1",
-    "@comunica/query-sparql": "^2.2.1",
-    "@comunica/runner": "^2.0.3",
+    "@comunica/config-query-sparql": "^2.2.0",
+    "@comunica/query-sparql": "^2.4.0",
+    "@comunica/runner": "^2.4.0",
     "@rubensworks/solid-client-authn-browser": "^1.13.0",
     "@turf/centroid": "^6.5.0",
     "babel-loader": "^8.2.3",
     "file-loader": "^6.0.0",
     "json-loader": "^0.5.7",
     "leaflet": "^1.7.1",
     "minimist": "^1.2.5",
-    "n3": "0.11.3",
-    "node-polyfill-webpack-plugin": "^1.1.4",
-    "rdf-string": "1.6.1",
+    "n3": "^1.16.2",
```

Comment:
Are you sure this doesn't break anything? Because there are breaking changes in this update.
Can probably be fixed using `rdf-string`.

There's a high chance that CONSTRUCT queries will be broken.

Reply:
Ohhh, `rdf-string` was still supposed to be there! My bad, I will add it back.

Reply:
No, I meant that the update of `n3` could cause breakages.
`rdf-string` is still present.

Reply:
I have added it back as a dependency, but some of the other packages depends on it already since it was included even with that entry removed. Thank you for noticing!

Reply:
So the `n3` does not cause any breakages? E.g. issuing CONSTRUCT queries works?

Reply:
That one I did not try, I will see what happens!

Reply:
CONSTRUCT queries do break with a missing `addTriple` function. I will try to see why.

Reply:
The issue with `n3` should be fixed now and CONSTRUCT queries work! But it collects all the quads into the writer and serializes everything at the end, which seems slightly problematic for large numbers of quads. At least it works, though. Apologies for not reading the earlier messages properly. :blush:

Reply:
> But it collects all the quads into the writer and serializes everything at the end, which seems slightly problematic for large numbers of quads.

Indeed, that's quite problematic.
Can you look into what will be needed to make it work in a streaming way as it did before?

For example, the following query should be able to produce a continuous stream of quads:
http://query.linkeddatafragments.org/#datasources=https%3A%2F%2Ffragments.dbpedia.org%2F2016-04%2Fen&query=CONSTRUCT%20WHERE%20%7B%0A%20%20%3Fs%20%3Fp%20%3Fo.%0A%7D

Reply:
I swapped the normal writer for a stream writer, but had to also add `process` override thing for readable-stream, or otherwise it would not find/bundle it for some reason. I did it in a way similar to this commit to Comunica: https://github.com/comunica/comunica/commit/cbf47cc0e38294d0fadb244fb9fab7e922db941c

But I had to use `typeof global.process` instead, since non-global `process` produced an error about a variable not being defined.

Now the query produces a (somewhat) continuous stream of quads.